ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+- [ ] Verify that your issue is not being currently addressed by other [open issues](https://github.com/scikit-learn/scikit-learn/issues) or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls).
+- [ ] If your issue is a usage question or does not potentially require changes to the codebase to be solved, then StackOverflow (using the `[scikit-learn]` tag) or our [mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general) may be a better place to bring it up. For more information, see ("User Questions")[https://lists.sourceforge.net/lists/listinfo/scikit-learn-general].
+
```

Comment:
Small nitpick: I think it should be ["User Questions"] (https.....). The brackets have been interchanged by mistake I think.